terraform/bosh-resources/main.tf

```diff
@@ -35,7 +35,7 @@ variable "baseip" {
 
 variable "bosh_cli_version" {
     type = "string"
-    default = "2.0.16"
+    default = "2.0.40"
```

Comment:
Is it possible to get this version from the environment variable?

Reply:
Environment variables override defaults anyway. The default is only used when the environment variable is not set, it acts as a fall back. If we omit the default line entirely, then failing to set the environment variable will result in a failure.

Also, IIRC, you cannot use other variables when defining a default